lib/rules/jsx-no-literals.js

```diff
@@ -40,27 +40,40 @@ module.exports = {
         },
         ignoreProps: {
           type: 'boolean'
+        },
+        noAttributeStrings: {
+          type: 'boolean'
         }
       },
+
       additionalProperties: false
     }]
   },
 
   create(context) {
-    const defaults = {noStrings: false, allowedStrings: [], ignoreProps: false};
+    const defaults = {
+      noStrings: false, allowedStrings: [], ignoreProps: false, noAttributeStrings: false
+    };
     const config = Object.assign({}, defaults, context.options[0] || {});
     config.allowedStrings = new Set(config.allowedStrings.map(trimIfString));
 
-    const message = config.noStrings
-      ? 'Strings not allowed in JSX files'
-      : 'Missing JSX expression container around literal string';
+    function defaultMessage() {
+      switch (true) {
+        case config.noAttributeStrings:
+          return 'Strings not allowed in attributes';
+        case config.noStrings:
+          return 'Strings not allowed in JSX files';
+        default:
+          return 'Missing JSX expression container around literal string';
+      }
+    }
 
     function reportLiteralNode(node, customMessage) {
-      const errorMessage = customMessage || message;
+      const errorMessage = customMessage || defaultMessage();
 
       context.report({
         node,
-        message: `${errorMessage}: “${context.getSourceCode().getText(node).trim()}”`
+        message: `${errorMessage}: "${context.getSourceCode().getText(node).trim()}"`
```

Comment:
the curly quotes here are quite intentional; straight quotes are never appropriate in prose. please revert this line.